wakuv2/common/filter.go

```diff
@@ -89,28 +98,34 @@ func (fs *Filters) Install(watcher *Filter) (string, error) {
 	}
 
 	watcher.id = id
+
 	fs.watchers[id] = watcher
 	fs.addTopicMatcher(watcher)
+
+	fs.logger.Info("Filters.Install()", zap.String("id", id))
 	return id, err
 }
 
 // Uninstall will remove a filter whose id has been specified from
 // the filter collection
 func (fs *Filters) Uninstall(id string) bool {
-	fs.mutex.Lock()
-	defer fs.mutex.Unlock()
-	if fs.watchers[id] != nil {
-		fs.removeFromTopicMatchers(fs.watchers[id])
+	fs.Lock()
+	defer fs.Unlock()
+	watcher := fs.watchers[id]
+	if watcher != nil {
+		fs.removeFromTopicMatchers(watcher)
 		delete(fs.watchers, id)
+
+		fs.logger.Info("Filters.Uninstall()", zap.String("id", id))
```

Comment:
Comments generally start from lower case, maybe Info is too high? debug?

Reply:
Fixed.